swift/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Use the latest Swift from Docker Hub
+FROM ibmcom/swift-ubuntu
+
+# Add current application
+ADD . .
```

Comment:
I'm just mirroring the other examples. Happy to change it, `.dockerignore` would be cleaner.